src/frequenz/sdk/actor/_power_managing/_bounds.py

```diff
@@ -138,7 +138,7 @@ def clamp_to_bounds(  # pylint: disable=too-many-return-statements
 
     # If the given value is within the exclusion bounds and the exclusion bounds are
     # within the given bounds, clamp the given value to the closest exclusion bound.
-    if exclusion_bounds is not None:
+    if exclusion_bounds is not None and not value.isclose(Power.zero()):
```

Comment:
If the power is close to zero, shouldn't we use `Power.zero()` instead of `value`? What happens if the API receives something that is *close to zero* but not exactly zero?

I understand in general we need to compare using *is close* comparison because of floats work, but if we treat zero specially we might run into problems too.